README.md

```diff
@@ -32,9 +34,60 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
 * [Generating portable installers](#generating-portable-installers)
 
-## Deployment
+## Quick Installation via Helm
 
-To quickly deploy the latest development version of StackRox to your kubernetes
+StackRox offers quick installation via Helm Charts, follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) for system specific instructions and requirements.
+First, add the [stackrox/helm-charts/opensource](https://github.com/stackrox/helm-charts/tree/main/opensource) repository to Helm.
+```sh
+helm repo add stackrox-oss https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+```
+To see all available Helm charts in the repo run (you may add the option `--devel` to show non-release builds as well)
+```sh
+helm search repo stackrox-oss
+```
+From here you can install stackrox-central-services to get Central and Scanner components deployed on your cluster. Note that you need only one deployed instance of stackrox-central-services even if you plan to secure multiple clusters.
+```sh
+helm install -n stackrox-oss --create-namespace stackrox-central-services stackrox-oss/stackrox-central-services --devel
+```
+To create a secured cluster, you first need to generate an init bundle containing initialization secrets. The init bundle will be saved in `stackrox-init-bundle.yaml`. Use it to provision secured clusters as shown below.
+```sh
+kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
+    --password "$(kubectl -n stackrox get secret central-htpasswd -o go-template='{{index .data "password" | base64decode}}')" \
+    central init-bundles generate stackrox-init-bundle --output - > stackrox-init-bundle.yaml
+```
+Then install stackrox-secured-cluster-services in the same cluster using this command with the init bundle you just generated:
+```sh
+helm install -n stackrox-oss --create-namespace stackrox-secured-cluster-services stackrox-oss/secured-cluster-services \
+    -f stackrox-init-bundle.yaml \ 
+    --set clusterName=<name_of_the_secured_cluster> 
+```
+Make sure to provide some name in `clusterName` argument meaningful to you. The cluster will be identified by this name in clusters list in StackRox UI.
+
+When deploying Secured Cluster Services on a different cluster, you will also need to specify the endpoint (address and port number) of Central via `--set centralEndpoint=<endpoint_of_central_service>` command-line argument.
+
+To further customize your Helm installation consult these documents:
+* <https://docs.openshift.com/acs/installing/installing_helm/install-helm-quick.html>
+* <https://docs.openshift.com/acs/installing/installing_helm/install-helm-customization.html>
+
+### First use
+
+Follow these steps to get to StackRox UI
+
+1. Obtain admin password:
+   `kubectl -n stackrox get secret central-htpasswd -o go-template='{{index .data "password" | base64decode}}'`
+
+2. Setup port forward to central:
+   `kubectl -n stackrox port-forward deploy/central 8000:8443`
```

Comment:
I suppose this renders not how you wanted it to look. See https://github.com/stackrox/stackrox/tree/jschnath/ROX-11172_Document_OSS_Helm_Chart_usage#first-use

I think you need to use triple backticks around code not single ones if you'd like code blocks to be rendered as blocks.

If you prefer the code to be rendered inline like how it is now, that's also fine by me.

----

Screenshot of the current rendering
![image](https://user-images.githubusercontent.com/537715/171587002-47f910c0-696c-441a-8dba-8fd7e0839c4f.png)

Reply:
backticks and triple backticks render the same in Goland, I'll put in triples here